importApp/src/main/java/com/commercetools/dataimport/orders/OrderCsvLineValue.java

```diff
@@ -0,0 +1,101 @@
+package com.commercetools.dataimport.orders;
+
+
+/**
+ * This class is used as a container for the csv line data
+ */
+
+public class OrderCsvLineValue {
+
+    private String customerEmail;
+    private String orderNumber;
+    private LineItems lineItems;
+    private double totalPrice;
+
+
+    public OrderCsvLineValue() {
+    }
+
+    public String getCustomerEmail() {
+        return customerEmail;
+    }
+
+    public void setCustomerEmail(String customerEmail) {
+        this.customerEmail = customerEmail;
+    }
+
+    public String getOrderNumber() {
+        return orderNumber;
+    }
+
+    public void setOrderNumber(String orderNumber) {
+        this.orderNumber = orderNumber;
+    }
+
+    public LineItems getLineItems() {
+        return lineItems;
+    }
+
+    public void setLineItems(LineItems lineItems) {
+        this.lineItems = lineItems;
+    }
+
+    public double getTotalPrice() {
+        return totalPrice;
+    }
+
+    public void setTotalPrice(double totalPrice) {
+        this.totalPrice = totalPrice;
+    }
+
+    public static class LineItems {
+
+        private Variant variant;
+        private double price;
+        private long quantity;
+
+        public double getPrice() {
+            return price;
+        }
+
+        public void setPrice(double price) {
+            this.price = price;
+        }
+
+        public long getQuantity() {
+            return quantity;
+        }
+
+        public void setQuantity(long quantity) {
+            this.quantity = quantity;
+        }
+
+
+        public Variant getVariant() {
+            return variant;
+        }
+
+        public void setVariant(Variant variant) {
+            this.variant = variant;
+        }
+
+        public static class Variant {
```

Comment:
I would have this class represented exactly as the CSV line, so without the inner classes (I guess @katmatt was referring to the same in the comment below?). I don't really see the need of it since it's only 3 more class fields you need to add. And actually having it separately you gain one field.
I'm assuming of course you did it to match the CTP structure (since I've checked the product CSV data class and it doesn't contain any inner classes), but if there's another reason then it might be worth it.